config/webpack/partial/manifest.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
```

Comment:
please name the file something else to avoid confusion.  maybe `pwa-manifest.js`.